src/sources/webgl.ts

```diff
@@ -133,15 +135,19 @@ export function getWebGlExtensions({ cache }: Options): WebGlExtensionsPayload |
   // Extension parameters
   if (extensions) {
     for (const name of extensions) {
+      const extension = gl.getExtension(name)
+      const extensionSupported = extension != null
+
+      supported.push(extensionSupported)
+
       if (
         (name === rendererInfoExtensionName && shouldAvoidDebugRendererInfo()) ||
         (name === polygonModeExtensionName && shouldAvoidPolygonModeExtensions())
       ) {
         continue
       }
 
-      const extension = gl.getExtension(name)
```

Comment:
This must be called after the above checks. Please read the comments for `shouldAvoidDebugRendererInfo` and `shouldAvoidPolygonModeExtensions` to know why. If you want to check `WEBGL_debug_renderer_info` and `WEBGL_polygon_mode` specifically, it's not doable.